examples/btc-to-evm-fungible-transfer/.env.sample

```diff
@@ -0,0 +1,10 @@
+SYGMA_ENV=testnet
```

Comment:
@wainola rename examples to `bitcoin-to-evm-fungbile-transfer` and `evm-to-bitcoin-fungible-transfer` for consistency